bluedonkey.service

```diff
@@ -3,6 +3,7 @@ Description=BlueDonkey Service
 After=multi-user.target
 
 [Service]
+ExecStartPre=/bin/sleep 90
```

Comment:
Probably makes more sense to use a timer service or to find out what the dependency is on startup. 

https://www.linux.com/tutorials/setting-timer-systemd-linux/